pyproject.toml

```diff
@@ -1,9 +1,9 @@
 [tool.poetry]
-name = "meltanolabs-tap-bigquery"
-version = "0.0.1"
+name = "tap-bigquery"
```

Comment:
Is the change in package necessary?

Reply:
No, I don't know why it was changed. I can undo it.

Reply:
Yeah we should probably revert. I don't think it hurts and will be one less change to make if we decide to publish to PyPI.

Reply:
https://github.com/MeltanoLabs/tap-bigquery/pull/24/commits/bfbcad79fa844ef055a31a3068f5814019a68f69